endpoints/bookstore-grpc/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM gcr.io/google_appengine/python
```

Comment:
Definitely link to the Python runtime page on github: https://github.com/GoogleCloudPlatform/python-runtime